src/.vuepress/theme/client/components/global/OpenApiServers.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <ul
+    v-if="servers?.length"
```

Comment:
je geeft in je props aan dat je altijd een servers element verwacht.

in je implementatie zie ik dat je default een `[]`.

Zo hebben we altijd `servers?`

Reply:
Dat klopt niet helemaal,

Het type is als volgt:
`(property) servers?: OpenApiType.ServerObject[] | undefined`

Ik kan daarom niet aannemen dat waar dit component ook geïmplementeerd wordt, servers niet ook `undefined` kan zijn.

Reply:
<img width="651" alt="Screenshot 2024-04-24 at 13 46 15" src="https://github.com/myparcelnl/developer/assets/2649958/12c46071-42c2-4380-925b-4247a35ca267">

Reply:
Ik doel op deze. Je stuurt nu als fallback een lege Array. Dus zal het nooit echt undefined worden. Right?